src/components/controllers/tutorial/index.js

```diff
@@ -278,8 +277,6 @@ function TutorialView({
 				}
 			>
 				<div class={style.tutorialWindow} ref={content}>
-					<h1 class={style.title}>{title}</h1>
```

Comment:
`style.title` doesn't actually exist so it didn't need to be cleaned up in the CSS.